src/index.d.ts

```diff
@@ -45,9 +45,16 @@ declare module 'slf' {
     (error: Error | null, event: Event): void;
   }
 
+  function ConsoleLogger(event: Event): void;
+
+  class Logger {
+    constructor(name: string, sink: Factory, chain: Middleware[]);
+    static getLogger(name: string): Log;
+  }
+
   class LoggerFactory {
-    static getLogger(name: string): Logger;
+    static getLogger(name: string): Log;
```

Comment:
Same as previous comment (referring to [this line](https://github.com/surikaterna/slf/blob/e16279f2e2346ebf54ff289815f2a1d6a71c7377/src/logger-factory.js#L32)).